pkgs/tools/package-management/appimage-run/default.nix

```diff
@@ -1,4 +1,10 @@
-{ appimageTools, buildFHSEnv, makeDesktopItem, extraPkgs ? pkgs: [], appimage-run-tests ? null }:
+{ lib
+, appimageTools
+, buildFHSEnv
+, makeDesktopItem
+, extraPkgs ? pkgs: []
+, appimage-run-tests ? null
+}:
```

Comment:
Nit: We'll have a big reformat soon anyways; there isn't much point in this.

Reply:
I don't think that is coming so soon, that we need to stop all efforts at making code readable.